charts/rstudio-pm/NEWS.md

```diff
@@ -1,3 +1,7 @@
+# 0.3.11
+
+- Add configuration values for the pod's `labels`, `sidecar`, `affinity`, `nodeSelector`, `tolerations`, and `priorityClassName` ([#206](https://github.com/rstudio/helm/issues/206)).
```

Comment:
Nit: we list `sidecar` here but no longer have a key for it after the rebase.

Reply:
🙈